src/client-tests/src/protractor.config.js

```diff
@@ -31,51 +32,56 @@ function getBrowserCapabilities() {
   }
 }
 
-exports.config = {
+const config = {
   allScriptsTimeout: 100000,
   framework: 'jasmine',
   directConnect: true,
   maxSessions: 10,
-  specs: ['tmp/*/*.js'],
+  specs: ['./e2e/**/*.spec.ts'],
   capabilities: getBrowserCapabilities(),
   jasmineNodeOpts: {
     isVerbose: true,
     realtimeFailure: true,
     showColors: true,
     includeStackTrace: true,
     defaultTimeoutInterval: 300000,
-    print: () => {},
+    print: () => { },
   },
-  onPrepare: function() {
-    setUpCustomLocators()
+  onPrepare: function () {
+    setUpCustomLocators();
+    require('ts-node').register()
```

Comment:
I added this to make both debugging easier, and the process simpler to just debug the ts-node files directly. 
This works really well in vs-code (and intellij) with the correct config. 
![debugging](https://user-images.githubusercontent.com/3878103/67190780-7e64ac80-f3e8-11e9-862d-525716d7a170.gif)